sdk/PrebidDemandSDKLoaders/AdServerCustomEvents/DFP/PrebidCustomEventInterstitialDFP.m

```diff
@@ -0,0 +1,119 @@
+/*   Copyright 2017 APPNEXUS INC
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "PBCacheLoader.h"
+#import "PBFacebookInterstitialAdLoader.h"
+#import "PrebidCustomEventInterstitialDFP.h"
+#import "PrebidMobileDemandSDKLoaderSettings.h"
+
+static NSString *const customEventErrorDomain = @"org.prebid.PrebidMobileMediationAdapter";
+
+@interface PrebidCustomEventInterstitialDFP()
+
+@property (strong, nonatomic) NSString *cacheId;
+@property (strong, nonatomic) NSString *bidder;
+@property (strong, nonatomic) PBBaseInterstitialAdLoader *adLoader;
+@property (strong, nonatomic) PBCacheLoader *cacheLoader;
+
+@end
+
+@implementation PrebidCustomEventInterstitialDFP
+
+@synthesize delegate;
```

Comment:
Is this delegate instantiated by DFP?

Reply:
yup its GADCustomEventInterstitialDelegate